CMakeLists.txt

```diff
@@ -230,8 +229,7 @@ message("\n")
 message("secp256k1 configure summary")
 message("===========================")
 message("Build artifacts:")
-message("  shared library ...................... ${SECP256K1_BUILD_SHARED}")
-message("  static library ...................... ${SECP256K1_BUILD_STATIC}")
+message("  shared library ...................... ${BUILD_SHARED_LIBS}")
```

Comment:
nit: When `BUILD_SHARED_LIBS` is FALSE/OFF, at first glance it is not obvious that the build system is configured to build a static library.